sda-commons-yaml/README.md

```diff
@@ -0,0 +1,78 @@
+# SDA Commons Shared YAML
```

Comment:
Do we really need this module? Isn't there something available from Spring? I would like to avoid maintaining more code than necessary.

Reply:
module removed

`com.fasterxml.jackson.dataformat.yaml.YAMLMapper` used in asyncapi module